.yaydoc.yml

```diff
@@ -0,0 +1,12 @@
+metadata:
+  author: FOSSASIA
+  projectname: Phimp.me
+  version: development
+build:
+  theme: sphinx_fossasia_theme
+  source: docs
+  logo: _static/logo.png
```

Comment:
It is not just about the start page. What about when we will have multiple pages?